Sources/PackageLoading/Module+PkgConfig.swift

```diff
@@ -78,7 +86,31 @@ private extension SystemPackageProvider {
         }
         return false
     }
-    
+
+    func pkgConfigSearchPath() -> AbsolutePath? {
+        switch self {
+        case .Brew(let name):
+            // Homebrew can have multiple versions of the same package. The
+            // user can choose another version than the latest by running
+            // ``brew switch NAME VERSION``, so we shouldn't assume to link
+            // to the latest version. Instead use the version as symlinked
+            // in /usr/local/opt/(NAME)/lib/pkgconfig.
+            let brewPrefix: String
+            if let prefix = getenv("BREW_PREFIX") {
```

Comment:
I used this before: 7b82fae1ec170e4ac6a66d96a5f0968bcacf8f03, how about using that?